applications/API/tests/test_forms.py

```
@@ -0,0 +1,136 @@
+"""
```

Pylint (4.0.1)

[error] Unrecognized option found: suggestion-mode

(E0015)